### DISCUSSION TITLE
New source: Holiday API

### DISCUSSION THREAD

**Discussion by imrehg** [airbytehq/airbyte#36108] [New Connector Request]
## Tell us about the new connector you’d like to have
Source for the [Holiday API](https://holidayapi.com).

## Describe the context around this new connector
This API is handy for all kinds of business settings, when combined with other internal sources of data. The sync is likely run relatively infrequently, since holidays don't change often. 

This is an API source connector ([docs](https://holidayapi.com/docs)), and likely the "holidays" endpoint is the only relevant one for AirByte.

This service is a freemium one, I'm using it with a free account currently for development, potentially paid if it works out.

## Describe the alternative you are considering or using

Rolling my own using their Python library https://github.com/holidayapi/holidayapi-python

## Are you willing to submit a PR?

Yes.

**Comment by btkcodedev** [comment]
@jrolom I would like to work on this issue

**Comment by btkcodedev** [comment]
Update:
Got the API keys,
Working on it

**Reply by imrehg** [reply]
How's it going so far? Still working on it, or would you like to put this task back to the queue for others to pick up?

**Reply by btkcodedev** [reply]
Working on it, Trying to incorporate new changes within airbyte

**Comment by natikgadzhi** [comment]
@btkcodedev ping me if you need any help, I'm happy to help too! 

It would be ideal to make this connector using _only_ connector builder itself. It looks possible. @imrehg, you only want the Holidays stream itself, right? 

UPD: Made a quick proof of concept here: https://gist.github.com/natikgadzhi/4749d33ecaa326b845b6840444d3c82b

UPD2: okay, first cut no edits: https://www.youtube.com/watch?v=tVVcKF1oM0c&t=4s

**Reply by btkcodedev** [reply]
Thanks, Sure thing! Will update soon

**Reply by natikgadzhi** [reply]
Recorded a short video with PoC, uploading.

**Comment by btkcodedev** [comment]
PR ref: https://github.com/airbytehq/airbyte/pull/36917

[a comment by an automated account is not shown]
